ansible/roles/couchdb2/tasks/main.yml

```diff
@@ -14,11 +14,26 @@
     user: "{{ couchdb2.username }}"
     password: "{{ couchdb2.password }}"
     force_basic_auth: true
-    status_code: 200, 409 # 409 means already set up (conflict)
+    status_code: 201, 409 # 409 means already set up (conflict)
     body: '{}'
   when: inventory_hostname == groups.couchdb2.0 and item != groups.couchdb2.0
   with_items: "{{ groups.couchdb2 }}"
 
+- name: Create system databases
+  uri:
+    url: "http{% if couchdb_secure %}s{% endif %}://{{ inventory_hostname|ipaddr }}:{{ couchdb_port }}/{{ item }}"
+    method: PUT
+    user: "{{ couchdb2.username }}"
+    password: "{{ couchdb2.password }}"
+    force_basic_auth: true
+    status_code: 201, 412 # 412 means already set up (conflict)
+    body: '{}'
+  when: inventory_hostname == groups.couchdb2.0
+  with_items:
+   - _users
+   - _replicator
+   - _global_changes
```

Comment:
Really, this is the solution?! Couch makes us create its built-in dbs for it? Some things about couch just still blow my mind...

Reply:
It does create them for you under some circumstances but when I ran the playbook they weren't created.

It seems like couch2.0.0 has only half implemented cluster setup functionality.